.github/workflows/test.yml

```diff
@@ -13,8 +13,8 @@ jobs:
   build_and_test_macOS:
     runs-on: macOS-latest
     steps:
-    - uses: ros-tooling/setup-ros@0.1.2
-    - uses: ros-tooling/action-ros-ci@0.1.0
+    - uses: ros-tooling/setup-ros@v0.1
+    - uses: ros-tooling/action-ros-ci@v0.1
```

Comment:
latest version for `action-ros-ci` is `v0.2`, but you'd also need to update the inputs here (either add `required-ros-distributions: foxy` to `setup-ros` or set `vcs-repo-file-url` for `action-ros-ci`).

Reply:
Thanks for the note - updated